lfs/config.go

```diff
@@ -126,6 +126,11 @@ func (c *Configuration) GitRemoteUrl(remote string, forpush bool) string {
 		return u
 	}
 
+	err := git.ValidateRemoteURL(remote)
```

Comment:
Could be usefull to convert `err := git.ValidateRemoteURL` in `remote, err := git.NormalizeRemoteURL`. If the user clones an url with a trailing `/`, es `git clone https://github/repo.git/` the `NewEndpointFromCloneURLWithConfig` [1] adds an unwanted `repo.git/.git/info/lfs` layer to the Endpoint that should be `repo.git/info/lfs`

[1] [lfs/endpoint.go#L41](https://github.com/github/git-lfs/blob/8f90f594162d0a7e90e2f5785287f4327f097235/lfs/endpoint.go#L41)